Tests/BacktraceTests/BacktraceTests.swift

```diff
@@ -12,14 +12,24 @@
 //
 //===----------------------------------------------------------------------===//
 
-@testable import Backtrace
 import XCTest
 
-final class BacktraceTests: XCTestCase {
-    func testExample() {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct
-        // results.
-        XCTAssert(true)
+public final class BacktraceTests: XCTestCase {
+    func testBacktrace() {
+        #if os(Linux)
+        let expectedError = UUID().uuidString
+        let pipe = Pipe()
+        let process = Process()
+        process.executableURL = URL(fileURLWithPath: ".build/release/Sample")
```

Comment:
This is an SPM implementation detail and not guaranteed to be stable, but it's the best we can do.

Reply:
Oh actually, could we `swift run Sample`?

Reply:
should we need the path for something (ie. if we can't use `swift run` for some reason), this might work:

```
$ swift build --show-bin-path
/Users/johannes/devel/swift-nio/.build/x86_64-apple-macosx/debug
$ swift build --show-bin-path -c release
/Users/johannes/devel/swift-nio/.build/x86_64-apple-macosx/release
```

Reply:
using swift run sounds like a great idea, I will try it!

Reply:
https://github.com/swift-server/swift-backtrace/pull/26/commits/720255753a0523324470b3db295d0a68f8ff5bc2